File: Java/swap.java
package Java;
import java.util.Scanner;

public class swap {
    public static void main(String[] args)
    {
        System.out.print("Enter the first number: ");
        Scanner sc = new Scanner(System.in);
        int num1 = sc.nextInt();
        System.out.print("Enter the second number: ");
        int num2 = sc.nextInt();
    
        int temp;
        temp = num1;
        num1 = num2;
        num2 = temp;
        System.out.println("First number after swapping is: "+ num1);
        System.out.println("Second number after swapping is: "+ num2);

    }    
}

File: C Programs/swap.c
#include<stdio.h>

int main()
{
    int num1, num2, temp;
    printf("Enter the first number: ");
    scanf("%d", &num1);
    printf("Enter the second number: ");
    scanf("%d", &num2);

    temp = num1;
    num1 = num2;
    num2 = temp;
    printf("First Number after Swapping is %d \n", num1);
    printf("Second Number after Swapping is %d \n", num2);

}